scripts/install-pandoc.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
```

Comment:
i'd check if `uname` == "darwin" (the bash appropriate way here) and exit if that is the case with a friendly message saying only intended for linux.